statscache_plugins/pagure/pagure_contribution.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+
+from statscache_plugins.pagure.utils import PagurePluginMixin, plugin_factory
+
+import sqlalchemy as sa
+
+
+class PluginMixin(PagurePluginMixin):
+    name = "Number of contribution to pagure.io"
+    summary = "Count the number of contribution to pagure.io per day"
+    description = """
+    Counts the number of messages on fedmsg related to pagure.io for each day.
+    """
+
+    _topics = ['io.pagure.prod.pagure.issue.comment.added',
```

Comment:
Might be easier to catch all topics and check in process() if the topic starts with 'io.pagure.prod' ?